### ISSUE TITLE
Single command for cloning and launch of Xcode for LoopWorkspace build, without using the script

### ISSUE THREAD

**Issue by bjornoleh** [LoopKit/loopdocs#291]
As a continuation of a discussion on FB today regarding request for translation of the build script.

@cyoung1024 requested translation of the build script, based on experience with french-speaking users, and @jonfawcett suggested using a few commands chained like this:

```
cd ~/Downloads
git clone --branch=master --recurse-submodules https://github.com/LoopKit/LoopWorkspace
cd ./LoopWorkspace
xed ./Loop.xcworkspace
```

The above works, but upon a second attempt, it will yield the error
`fatal: destination path 'LoopWorkspace' already exists and is not an empty directory.
`

Below is a possible solution, heavily inspired by the build script 
(naming the folder by date-hour-minute-**second** might be excessive, but was useful during testing...)

```
cd ~/Downloads
mkdir ~/Downloads/Loop-$(date +'%y%m%d-%H%M%S')
cd ~/Downloads/Loop-$(date +'%y%m%d-%H%M%S')
git clone --branch=master --recurse-submodules https://github.com/LoopKit/LoopWorkspace
cd ./LoopWorkspace
xed .
```

If attaching a "Copy-button" to the above, there is really no need for the script during building of Loop. And the need for translation will also be gone. And the utility script commands are already described in Loopdocs for execution by command line.

The script is just fine, of course, but apparently translation is an issue there.

**Comment by bjornoleh** [comment]
I'm just wondering if the above will work for Macs of all language set ups, is there always a Downloads folder? I guess we could make one of not?

**Comment by cyoung1024** [comment]
> I'm just wondering if the above will work for Macs of all language set ups, is there always a Downloads folder? I guess we could make one if not?

There's always a downloads folder, and while its display name is different when looking at it in Finder it's always "downloads" in shell, so no worries there

**Comment by marionbarker** [comment]
Closed with PR #293
